sdk/tests/conformance/extensions/webgl-draw-buffers.html

```diff
@@ -228,8 +228,8 @@
 
   debug("Testing drawBuffersWEBGL with default drawing buffer");
   shouldBe("gl.getParameter(ext.DRAW_BUFFER0_WEBGL)", "gl.BACK");
-  wtu.shouldGenerateGLError(gl, gl.INVALID_VALUE, "ext.drawBuffersWEBGL([])");
-  wtu.shouldGenerateGLError(gl, gl.INVALID_VALUE, "ext.drawBuffersWEBGL([gl.NONE, gl.NONE])");
+  wtu.shouldGenerateGLErrorIn(gl, [gl.INVALID_VALUE, gl.INVALID_OPERATION], "ext.drawBuffersWEBGL([])");
```

Comment:
It looks to me like this test has always been buggy, and that it should have produced INVALID_OPERATION, not INVALID_VALUE, since the length of the incoming sequence is 0 (corresponding to the n==0 case in the EXT_draw_buffers spec). Do you agree?